server/config/routes.js

```diff
@@ -640,4 +642,18 @@ exports.configure = function configure(app) {
   app.post('/install', install.proceedInstall);
 
   app.get('/diagnoses', diagnoses.list);
-};
+
+
+  // roles
+  app.get('/roles/project/:project_id', rolesCtrl.list);
+  app.get('/roles/:uuid', rolesCtrl.detail);
+  app.post('/roles', rolesCtrl.create);
+  app.put('/roles/:uuid', rolesCtrl.update);
+  app.delete('/roles/:uuid', rolesCtrl.remove);
+  app.post('/roles/affect/units', rolesCtrl.affectPages);
```

Comment:
Instead of `/roles/affect/units/`, it would be better to follow the same syntax as `permissions`.  For assigning permissions to a user, we use `POST /users/:id/permissions`.  This is read "assign to user id `:id` the following permissions".

So, here you could do `app.post(/roles/:uuid/units', roles.affectPages)`.  This would be read "assign the following units to the role with uuid `:uuid`".